gradle/elide.versions.toml

```diff
@@ -51,7 +51,7 @@ cyclonedx = "2.0.0"
 dekorate = "3.1.1"
 dependencyAnalysis = "2.8.0"
 detekt = "1.23.8"
-dirs = "bf76af41b9"
+dirs = "26"
```

Comment:
this override was in place because we needed to fork the lib to build with updated deps (i think?). anyway, it's probably worth keeping, or figuring out why we forked and making a decision based on that (if we forked to add features, are those features worth keeping? for fixes, are the fixes still needed?)